sphinxmixcrypto/client.py

```diff
@@ -35,6 +35,24 @@ def destination_encode(dest):
 
 
 def create_header(params, route, pki, dest, message_id, rand_reader):
+    """
+    Create a sphinx header, used to construct forward messages and reply blocks.
+
+    :param params: An instance of SphinxParams.
+
+    :param route: A list of 16 byte mix node IDs.
+
+    :param pki: An implementation of IMixPKI.
+
+    :param dest: A "prefix free encoded" destination type or client ID.
```

Comment:
the prefix free encoding is the ugliest most confusing part of this API. i should exterminate it... but not in this PR.